README.md

```diff
@@ -37,6 +37,22 @@ idx(props, _ => _.user.friends[0].friends)
 The second argument must be a function that returns one or more nested member
 expressions. Any other expression has undefined behavior.
 
+## Flow Type
+
+[Flow](https://flow.org/) understands the `idx` idiom:
+
+```javascript
+// @flow
+
+import idx from 'idx';
+
+function getName(props: User) {
```

Comment:
Could you have also made the return type `?string` instead of adding line 53?

Reply:
I wanted to show flow's inference powers, but sure!